lighthouse-core/config/constants.js

```diff
@@ -39,7 +39,7 @@ const defaultSettings = {
 
   // the following settings have no defaults but we still want ensure that `key in settings`
   // in config will work in a typechecked way
-  locale: null, // default determined by the intl library
+  locale: 'en-US', // actual default determined by Config using lib/i18n
```

Comment:
since `Config.initSettings` will never use this, what is it used for

Reply:
> since Config.initSettings will never use this, what is it used for

since we declare the constants in this file to be a valid `Config.Settings`, unfortunately we have to give *some* valid value here. Unfortunately I can't think of a good way to not do that while still retaining all the benefits we currently have of the type checked `Settings` and making sure every settings property that needs to be defined has a valid value.